app/controllers/users/my_avatar_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Users::MyAvatarController < ApplicationController
```

Comment:
Should stick with the `My::AvatarsController` format we have for the rest of the `/my` namespace.